app.py

```diff
@@ -54,9 +104,30 @@ def search():
 def knn_search():
```

Comment:
**[Required, Merge conflict]** You'll need to add in the annotations for the exception handler here. Something like:

```python
@app.route('/search')
@cross_origin()
@ErrorHandlerWrapper()
@RateLimiter()
def knn_search():
    ...
```